compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -315,6 +323,25 @@ class TreeUnpickler(reader: TastyReader,
             tp.withVariances(vs)
           case _ => tp
 
+        def readModifiers(modReader: TreeReader, end: Addr): FlagSet = {
+          import modReader.reader._
+          var mods = EmptyFlags
+          while (currentAddr.index < end.index) {
+            readByte() match {
+              case IMPLICIT => mods |= Implicit
+              case ERASED   => mods |= Erased
+              case GIVEN    => mods |= Given
+              case tag => pickling.println(s"ignoring ``modifier``: ${astTagToString(tag)}")
+            }
+          }
+          verifyMethodTypeFlags(
```

Comment:
Same here. We'll fail anyway since wrong flags will give us a miss when we try to find a companion. So no need to check this here.